src/test/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriodTest.java

```diff
@@ -71,8 +73,14 @@ public void testTotalMonths() {
      *  TODO fix this test
      */
     @Test
-    public void testHashExpired() {
-        fail("This test needs to written!.");
+    public void testHasExpired() {
+       /**  fail("This test needs to written!."); */
+        SubscriptionPeriod subscriptionPeriod = new SubscriptionPeriod(now.getTime(), sixthMonthsFromNow.getTime());
+
+        boolean has_it_Expired = subscriptionPeriod.hasExpired(now.getTime());
+        boolean expired = true;
+
+        assertEquals(expired,  has_it_Expired);
     }
 
```

Comment:
use the version of the assert method that takes an initial string. that string should be a diagnostic message. 
e.g.
assertTrue("verify hasExpired is correct). 

Also, use camel case formatting rather than underscores in variable names.